src/app/courses/courses.component.ts

```diff
@@ -145,6 +148,10 @@ export class CoursesComponent implements OnInit, OnChanges, AfterViewInit, OnDes
     this.dialogsLoadingService.start();
   }
 
+  @HostListener('window:resize') OnResize() {
+    this.isMobileView = this.checkMobileService.checkIsMobile();
```

Comment:
These needs to be updated later on with new devicetype

Reply:
Yes, WIll put the PR on hold for now